vue/sbc-common-components/src/components/SbcSystemErrorModal.vue

```diff
@@ -6,7 +6,7 @@
 
 <script lang="ts">
 import { Vue, Component, Prop, Emit } from 'vue-property-decorator'
-import SbcSystemError from '@/components/SbcSystemError.vue'
+import SbcSystemError from '../components/SbcSystemError.vue'
```

Comment:
Any particular reason we are setting all these to relative paths?  There is a `tsconfig` entry to allow us to use the `@` path...

Reply:
`@` works fine when the project is run by itself, but when it's a package and its files are imported into a parent project, `@` points to the wrong place. This works around that.